iidm/iidm-reducer/src/main/java/com/powsybl/iidm/reducer/DefaultNetworkFilter.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.reducer;
+
+import com.powsybl.iidm.network.*;
+
+/**
+ * A default implementation of {@link NetworkFilter} that keeps everything.
+ *
+ * @author Mathieu Bague <mathieu.bague at rte-france.com>
+ */
+public class DefaultNetworkFilter implements NetworkFilter {
+
+    @Override
+    public boolean accept(Substation substation) {
+        return true;
+    }
+
+    @Override
+    public boolean accept(VoltageLevel voltageLevel) {
+        return true;
+    }
+
+    @Override
+    public boolean accept(Line line) {
```

Comment:
Is it usefull to have lines and transfos in this interface as it is deduced from voltage level and substation filtering?

Reply:
One of the first ideas we get, was to select identifiables (even Line, Transformers and HVDC line) to keep by name. But you’re right, in the default implementation, we deduced them from the substation and the voltageLevel. And I don’t know if it will done like this one day.